code/datums/components/mechComp_signals.dm

```diff
@@ -289,9 +308,7 @@
 					return COMSIGBIT_ATTACKBY_COMPLETE
 				if(DC_ALL)
 					WipeConnections()
-					if(istype(parent, /atom))
-						var/atom/AP = parent
-						boutput(user, "<span class='notice'>You disconnect [AP.name].</span>")
+					boutput(user, "<span class='notice'>You disconnect [src].</span>")
```

Comment:
Same here, The `This is wrong, my old comment shouldn't be here. I suspect a faulty merge-conflict resolution` Should be kept, and the old `[src] ` removed.